client/js/components/Favourites.jsx

```diff
@@ -82,22 +92,25 @@ class Favourites extends React.Component {
       );
     }
     return (
-      <div className="">
+      <div>
         <Navbar />
```

Comment:
Why not render the Navbar in the parent component so that all pages have it instead of rendering it in each individual component?

Reply:
fixed. thanks